tests/__init__.py

```diff
@@ -20,6 +20,9 @@
 )
 PG_VERSIONS = frozenset(environ.get("PG_VERSIONS", "12").split())
 SCAFFOLDINGS_DIR = join(DIR, "scaffoldings")
+GEIOP_CREDENTIALS_PROVIDED = environ.get("GEOIP_LICENSE_KEY", False) and environ.get(
```

Comment:
```suggestion
GEIOP_CREDENTIALS_PROVIDED = environ.get("GEOIP_LICENSE_KEY") and environ.get(
```

Reply:
I prefer to have explicit return values for dict get to avoid type confusion bugs with implicit return values. Even if it wouldn't be needed here.